src/editor/package.json

```diff
@@ -11,13 +11,13 @@
     "graphiql": "0.13.2",
     "graphql": "14.5.0",
     "marked": "0.3.6",
-    "react": "15.6.2",
-    "react-dom": "15.6.2"
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
```

Comment:
@mac2000 One note please keep exact versions in `package.json` it helps a lot with debugging dependencies.

Reply:
Got it, thank you